modules/opensearch/includes/opensearch.search.inc

```diff
@@ -132,13 +134,47 @@ function opensearch_search_search(TingSearchRequest $ting_query) {
     $query = '';
   }
 
-  // TODO: Handle sorts.
-  // Perform the Open Search search and wrap the result.
+  // Prepare options.
   $options = [];
+
+  // Set sort options.
+  $field_mapping = _opensearch_search_map_common_sort_fields();
+  if (!empty($ting_query->getSorts())) {
+    // First map sort instances to opensearch compatible search strings.
+    $mapper = function(TingSearchSort $instance) use ($field_mapping) {
+      if ($instance->getDirection() === TingSearchSort::DIRECTION_ASCENDING) {
+        $direction = '_ascending';
+      }
+      elseif ($instance->getDirection() === TingSearchSort::DIRECTION_DESCENDING) {
+        $direction = '_descending';
+      }
+      else {
+        $direction = '';
+      }
+
+      // Map from common field back to our sort field.
+      if (isset($field_mapping[$instance->getField()])) {
+        $field = $field_mapping[$instance->getField()];
+      }
+      else {
+        // Not a common field, so we can assume the field has been provided by
+        // opensearch_search_sort_options so no need to map it.
+        $field = $instance->getField();
+      }
+
+      return $field . $direction;
+    };
+
+    // Then if we have mutiple sorts seperate them by space.
+    $prepared_sorts = array_map($mapper, $ting_query->getSorts());
+    // We currently only support a single sort.
+    $options['sort'] = $prepared_sorts[0];
```

Comment:
It is a little confusing that the first comment mentions multiple sorts and the following says that we only support one.

Perhaps `reset()` or `array_shift()` to get the initial value.